qcodes/dataset/data_set.py

```diff
@@ -567,26 +567,32 @@ def get_data(self,
                  start: Optional[int] = None,
                  end: Optional[int] = None) -> List[List[Any]]:
         """ Returns the values stored in the DataSet for the specified parameters.
-        The values are returned as a list of parallel NumPy arrays, one array
-        per parameter. The data type of each array is based on the data type
-        provided when the DataSet was created. The parameter list may contain
-        a mix of string parameter names, QCoDeS Parameter objects, and
+        The values are returned as a list of lists, rows by columns, e.g.
+        datapoint by parameter. The data type of each element is based on the
```

Comment:
I still did not understand the format. for an sql table like this:

x | y | z
--|---|--
1 | 2 | 3
4 | 5 | 6

is it going to return:
```python
[ [1, 2, 3],
  [4, 5, 6] ]
```
?